avm/src/lib.rs

```diff
@@ -55,11 +55,12 @@ pub fn use_version(version: &Version) -> Result<()> {
                 "anchor-cli {} is not installed, would you like to install it? (y/n)",
                 version
             ))
-            .with_initial_text("y")
             .default("n".into())
             .interact_text()?;
         if matches!(input.as_str(), "y" | "yy" | "Y" | "yes" | "Yes") {
             install_version(version)?;
+        } else {
+            return Err(anyhow!("Version {} is not installed", version));
```

Comment:
what do you think about not making this an error but just a print (like `User chose not to install version <new_version>. Staying on version <old_version>.`) + return?
The command behaved as expected so I wouldnt consider it an error

Reply:
Agreed, that is nicer